GSA/digitalgov.gov
# Topics Removal: 3+ topics

## Summary

This PR will removed 3 or more topics that are needed. 

URLs have been added to Wayback Machine as well.

New topics number from 491 → 462.

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/cm-3More-topics-removal/)



### Solution
remove both the topic along with where they are tagged throughout the site. 

### How To Test

1. Refer to the spreadsheet within the ticket titled `DG - Topics: Remove ROT topics` ensure that all files have been removed

---

### Dev Checklist
